src/apollo/utils/Secp256k1PrivateKey.ts

```diff
@@ -26,7 +26,22 @@ export class Secp256k1PrivateKey
   public type: KeyTypes = KeyTypes.EC;
 
   public readonly to = ExportableKey.factory(this, { pemLabel: "EC PRIVATE KEY" });
-  static from = ImportableKey.factory(Secp256k1PrivateKey, { pemLabel: "EC PRIVATE KEY" });
+  static from = {
+    ...ImportableKey.factory(Secp256k1PrivateKey, { pemLabel: "EC PRIVATE KEY" }),
+    Seed: (seedHex: string) => {
+      const seed = Buffer.from(seedHex, "hex");
+      const baseKey = HDKey.InitFromSeed(Int8Array.from(seed), 0, BigIntegerWrapper.initFromInt(0));
+
+      if (baseKey.privateKey == null) {
+        throw new ApolloError.MissingPrivateKey();
+      }
+
+      const key = new Secp256k1PrivateKey(Buffer.from(baseKey.privateKey));
+      key.keySpecification.set(KeyProperties.seed, seedHex);
```

Comment:
Uhn I don't think we should do this. Keeping the derivation path in the keyProperties is acceptable, but having the seed in the key specification is not that can lead to leaks.

Reply:
seed is no longer set on the instance